reporting/html.go

```diff
@@ -0,0 +1,37 @@
+package reporting
+
+import (
+	"bytes"
+	_ "embed"
+	"html/template"
+	"log"
+)
+
+var (
+	//go:embed html/report.tmpl
+	htmlTemplate string
+	//go:embed html/report.css
+	cssTemplate string
+	//go:embed html/github.svg
+	githubSVG string
+	//go:embed html/checkmarx_logo.html
+	checkmarxLogo string
+)
+
+func writeHtml(report Report) string {
+	tmpl := template.Must(template.New("report").Funcs(getFuncMap()).Parse(htmlTemplate))
```

Comment:
It is new for me.

When the `template.New("report").Funcs(funcMap).Parse(htmlTemplate)` returns `(*template.Template, error)`, the `template.Must` that accept `*template.Template, error` will get the multi-var return value?
Good to know!